diagnostic.md

```diff
@@ -35,19 +35,19 @@ Open up this file in Atom, and write your answers below (where indicated).
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+This is an absolute path because it begins with a /, which indicates the root directory is the first level of this path.
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+We would use a relative path to refer to the font because we do not have the file for the font downloaded onto our machines.  It would have to reference a website instead.
```

Comment:
Absolute. When you have to access something from the internet, you need the absolute path in order to retrieve the information.